site/viamguides/src/ai-camera-tracking/ai-camera-tracking.md

```diff
@@ -0,0 +1,423 @@
+1. Notice adding this module adds the board hardware component called `board-1`. The collapsible card on the right corresponds to the part listed in the left sidebar.
+   ![added board](assets/addBoard.webp)
+1. Click **Save** in the top right to save and apply your configuration changes.
+
+### Configure your pan servo
+
+1. In [the Viam app](https://app.viam.com/fleet/locations), find the **CONFIGURE** tab.
+1. Click the **+** icon in the left-hand menu and select **Component**.
+1. Select servo, and find the `raspberry-pi:rpi-servo` module. This adds the module for controlling a servo with a Raspberry Pi's GPIO pins. Set the name to `pan` to help indicate the responsibility of this servo.
+   ![add servo component](assets/addPanServo.webp)
+1. Notice adding this module adds the servo hardware component called `pan`. The collapsible card on the right corresponds to the part listed in the left sidebar.
+1. In the new `pan` panel, configure your component with the following JSON in the **CONFIGURE** field. This tells your servo component to use a specific pin on a specific board (called `board-1` in the Viam app).
+   ```json
+   {
+       "board": "board-1",
+       "pin": "8"
+   }
+   ```
+   ![configure pan servo](assets/configurePanServo.webp)
+1. Click **Save** in the top right to save and apply your configuration changes.
+1. At the bottom of the `pan` panel, expand the **TEST** section to ensure you have configured the servo correctly. You should see the servo move the camera mount left and right using the **Quick move** controls.
+   ![test servo](assets/testServo.webp)
+
+### Configure your tilt servo
+
+1. In [the Viam app](https://app.viam.com/fleet/locations), find the **CONFIGURE** tab.
+1. Click on the **...** in the top-right corner of the `pan` servo panel to reveal an action menu. Select **Duplicate** to create a copy of the component for the tilt servo in the configuration.
+   ![duplicate servo component](assets/duplicateServo.webp)
+1. A new component card will appear with the name of the `pan` servo component plus "-copy". Click on the name and rename it to `tilt`.
+   ![update tilt servo name](assets/updateTiltServoName.webp)
+1. In the new `tilt` panel, configure your component with the following JSON in the **CONFIGURE** field. This tells your servo component to use a specific pin on a specific board (called `board-1` in the Viam app).
+   ```json
+   {
+       "board": "board-1",
+       "pin": "10"
+   }
+   ```
+   ![configure tilt servo](assets/configureTiltServo.webp)
+1. Click **Save** in the top right to save and apply your configuration changes.
+1. At the bottom of the `tilt` panel, expand the **TEST** section to ensure you have configured the servo correctly. You should see the servo move the camera mount up and down using the **Quick move** controls.
+   ![test servo](assets/testServo.webp)
+
+### Configure your webcam
```

Comment:
Consider moving the entire "Configure your webcam" step to right after "Configure your Raspberry Pi board" but before the "Configure your pan servo" and "Configure your tilt servo" steps. In this way, the screenshots match what's currently been set up. It also takes away the potential panic folks might feel when they see a "camera-1" in one of the screenshots during the configuring of the pan servo (but they still haven't set it up yet because the camera step is a bit further down)